### ISSUE TITLE
[1.16 Suggestion] Elenai Dodge 2 Support

### ISSUE THREAD

**Issue by NovaN4** [pau101/Wings#76]
A little early to be suggesting this, but after the mod ports I'd love to see Elenai Dodge 2 support. They add a feather bar for their dodge system that I think could be utilised by this mod's flight system. I basically propose a config option that makes flight drain feathers rather than hunger; this could make flying with full armor much harder compared to flying with no armor or light armor.

**Comment by NovaN4** [comment]
Indeed, but the feather bar drain speed should be configurable just like the hunger drain speed currently is.

**Comment by Frey74** [comment]
And also, heavy armour like Iron should also make the wings slower.
